js/reference.js

```diff
@@ -377,70 +372,52 @@ var ERMrest = (function(module) {
 
                 var defer = module._q.defer();
 
-                var uri = this._uri;
+                var uri = this._location.compactUri;
```

Comment:
Nevermind, I think I'm confused here because I'm looking at the diff and cannot see enough of the surrounding code. Scratch this coment.